string_lab.js

```diff
@@ -0,0 +1,197 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+//
+//
+let str = 'Hello Stephanie'
+let firstHalf = str.slice(0,5)
+let secondHalf = str.slice(6, str.length)
+console.log(firstHalf)
+
+//
+// 2. `console.log` the length of a string.
+let name = 'Stephanie'
+console.log(name.length)
+
+
+// // 3. `console.log` the last character of any string.
+let steph= 'Good morning Stephanie'
+let  firstword = steph.slice(0,5)
+let middleword = steph.slice(5,12)
+let lastword = steph.slice(13,steph.length)
+console.log(lastword)
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+
+let nameB = 'estefanie'
+let withoutLastChar = nameB.slice(0,nameB.length-1)
+let lastCapitalized = nameB[nameB.length-1].toUpperCase()
+console.log(withoutLastChar + lastCapitalized)
+
+
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+// practice:
+// let x = 1000000
+// let variableAmount =  ' $ ' + x
+//
+// if (x === 1000000); {
+//   console.log(variableAmount + 'dollars' +' (pinky)');
+// } if (x === 1); {
+//   console.log(variableAmount) + 'dollar';
+// } if (x === 0) {
+// console.log('no' + variableAmount + 'dollar')
+// } else if {
+// console.log(variableAmount' + 'dollars');
+// }
+//
+// answer:
```

Comment:
Actually your practice was the right answer. Your actual answer isn't what this problem was asking.